src/metabase/util/embed.clj

```diff
@@ -124,3 +124,10 @@
              (if-not (and config/ee-available? (:valid (premium-features/token-status)))
                (setting/get-value-of-type :boolean :show-static-embed-terms)
                false)))
+
+(defsetting show-sdk-embed-terms
+  (deferred-tru "Check if admin should see the SDK licensing terms popup")
+  :type    :boolean
+  :visibility :public
```

Comment:
should this be scoped down to admins only?
I think there were some discussions about having the least possible public settings

Reply:
That'd behave the same as `show-static-embed-terms` 
https://github.com/metabase/metabase/blob/a2794d6ae23755cc72dec4fb4ba943b5556fa5f5/src/metabase/util/embed.clj#L118-L126

as no `:visibility` should default to `:admin`
https://github.com/metabase/metabase/blob/a2794d6ae23755cc72dec4fb4ba943b5556fa5f5/src/metabase/models/setting.clj#L1010